src/main/thrift/parquet.thrift

```diff
@@ -881,15 +922,21 @@ struct ColumnChunk {
   /** Crypto metadata of encrypted columns **/
   8: optional ColumnCryptoMetaData crypto_metadata
 
-  /** Encrypted column metadata for this chunk **/
+  /** Encrypted column metadata for this chunk 
+    *  
+    * PAR3: Not set see column_metadata_page on FileMetadata struct
```

Comment:
We can move (copy) the field 8 (`optional ColumnCryptoMetaData crypto_metadata`) to a V3 per column metadata structure that applies to all row groups / chunks. The field 9 (`optional binary encrypted_column_metadata`) is simply the encrypted version of field 3 (`optional ColumnMetaData meta_data`) - to hide the stats - so should be set per chunk (row group).

Reply:
I'm actually moving this back here (doing a pretty large refactor, would you mind seeing if this comment still applies?

Reply:
sure, will do after the refactoring.

Reply:
OK, done for now.

Reply:
I couldn't find a  `column_metadata_page` field in the updated FileMetaData struct.
Also, in the ColumnChunk struct, we still keep unencrypted ColumnMetaData (field 3:). Why would we move its encrypted version to a different struct?

Reply:
probably the comment change is not updated?

Reply:
Yes, missed this will update the comment.

Reply:
should be fixed now (removed PAR3 comment).

Reply:
Thanks, lgtm.